src/key_value_store/kv_flat_btree_async.cc

```diff
@@ -1120,7 +1120,7 @@ int KvFlatBtreeAsync::cleanup(const index_data &idata, const int &error) {
     //all changes were created except for updating the index and possibly
     //deleting the objects. roll forward.
     vector<pair<pair<int, string>, librados::ObjectWriteOperation*> > ops;
-    librados::ObjectWriteOperation owos[idata.to_delete.size() + 1];
+    vector<librados::ObjectWriteOperation> owos(idata.to_delete.size() + 1);
```

Comment:
If we really wanted we could make a variable-length fixed-size array (i.e. its size is determined by runtime) that lives on the stack and can hold objects like this, but that's a task for another time.